### PR TITLE
Fix change step when stepsItem is not present

### PR DESCRIPTION
**Pull request by re2005**
Fixes #

## Proposed Changes

- When using the steps inside pagination, prevent error.
-
-

### CODE REVIEW

File: src/components/steps/Steps.vue [hunk 1]
```diff
@@ -199,7 +199,7 @@ export default {
         * Change the active step and emit change event.
         */
         changeStep(newIndex) {
-            if (this.activeStep === newIndex) return
+            if (this.activeStep === newIndex || this.stepItems[newIndex]) return
```
Comment (service-paradis) [inline src/components/steps/Steps.vue]: I'm not sure this would keep the correct behavior...

Comment (re2005) [reply src/components/steps/Steps.vue]: I'm also not 100% sure.
The use case is: Having the steps inside pagination, when you go to page two, seems that the stepsArray is empty and "this.stepItems[newIndex].activate()" will fail.
It seems save to escape if there are no stepsItems.

Comment (service-paradis) [reply src/components/steps/Steps.vue]: But right now, the function exit when the newindex exists in the stepItems. So the code below would never run.

The condition to return should probably be something like `newIndex < this.stepItems.length` instead. So the function will return early only if the new index is not part of the step items.

Comment (re2005) [reply src/components/steps/Steps.vue]: You're totally right **newIndex** can never be bigger than the total index.
I've changed the PR and also added error message.
Thanks

**Comment by jtommy** [comment]
Thanks

**Comment by jtommy** [comment]
@re2005 now a test is failing, please check it

**Pull request closed without merging**